purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -462,14 +456,15 @@ class Purchases internal constructor(
         )
     }
 
+    // TODOBC5: Remove storeProduct parameter
     /**
      * Purchase a [StoreProduct]'s [PurchaseOption].
      * @param [activity] Current activity
      * @param [storeProduct] The StoreProduct of the product you wish to purchase
      * @param [purchaseOption] Your choice of purchase options available for the StoreProduct
```

Comment:
maybe we can update this to be explicit yet again, to "Your choice of purchase options available for the subscription `StoreProduct`